src/app/layouts/bootstrap-uploader/bootstrap-uploader.component.ts

```diff
@@ -38,6 +42,11 @@ export class BootstrapUploaderComponent {
                     });
                 }
 
+                this.googleAnalyticsEventsService.emitEvent(
+                    'ThemeBuilder',
```

Comment:
It seems that all event will have this category (ThemeBuilder). Can we remove the first argument of the function emitEvent and use the string ThemeBuilder inside it.